goblin_test.go

```diff
@@ -433,3 +435,48 @@ func TestItTimeout(t *testing.T) {
 		t.Fatal("Failed")
 	}
 }
+
+func TestItRegexExec(t *testing.T) {
```

Comment:
I definitely don't like this. We can also use tescript (https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript?tab=doc) if we're planning to do something like this. But I'd avoid having to use exec at all costs.